### PR TITLE
Reset overlay settings

### PR DESCRIPTION
**Pull request by inderpreetsingh**
Reset overlay settings button - that brings all the settings to default.

### CODE REVIEW

File: www/js/overlay.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
+/* eslint-disable import/no-unresolved */
 const electron = require('electron');
 const h = require('hyperscript');
 const ip = require('ip');
 const copy = require('copy-to-clipboard');
+const { obs: defaultPrefs } = require('./js/defaults.json');
```
Comment (tsingh777) [inline www/js/overlay.js]: is this path correct? this file is in the same directory as `defaults.json`

Comment (inderpreetsingh) [reply www/js/overlay.js]: Yes, the paths are referenced from `www` folder. So `./defaults.json` would give an error.

**Comment by tsingh777** [comment]
this will probably have conflicts with https://github.com/KhalisFoundation/sttm-desktop/pull/1016 so lets get that in first?

File: www/js/overlay.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
+/* eslint-disable import/no-unresolved */
```
Comment (tsingh777) [inline www/js/overlay.js]: prefer if we do `// eslint-disable-next-line import/no-unresolved` before the line

**Pull request merged**